docker-compose.yaml

```diff
@@ -121,7 +121,7 @@ services:
   event-listener:
     user: "${UID}:${GID}"
     container_name: event-listener
-    image: ghcr.io/scilifelabdatacentre/serve-event-listener/event-listener:v0.1.6
+    image: ghcr.io/scilifelabdatacentre/serve-event-listener/event-listener:v0.1.7
```

Comment:
I guess this needs an update as well
https://github.com/ScilifelabDataCentre/serve-charts/blob/2940c38422f1e402129e0f3cdbcf4e50b4899c54/serve/values.yaml#L273

Reply:
Yes, that change is part of this PR